client/src/http/http.js

```diff
@@ -0,0 +1,8 @@
+import axios from "axios";
+import { config as baseConfig } from "./config";
```

Comment:
Try to setup axios by creating instance with needed parameters, so we would not create configuration for each request.

Reply:
Done